### DISCUSSION TITLE
Context is undefined

### DISCUSSION THREAD

**Discussion by mhyfritz** [solidjs/solid#690] [Q&A]
I am playing with @ryansolid's nice WebGL/three.js [Sandbox](https://codesandbox.io/s/lmpei) and wanted to add a Context so that children have access to things like the camera.

However this does not work (Context in children is undefined) which probably is because the children are actually not explicitly passed through in the JSX of the `<Scene>` component.

I have created a [forked Sandbox](https://codesandbox.io/s/solid-gl-boxes-forked-iis3n?file=/solid-three.js). The Context is provided by `<Scene>` and I am accessing it inside `<Mesh>` (where it is undefined).

Any idea how to make this work? Thanks!

**Comment by high1** [comment]
It will work if you wrap the Scene itself with that context, the children are not wrapped in it
```
<SceneContext>
  <Scene>
      <PerspectiveCamera location={[0, 0, 3.2]} />
        <DirectionalLight direction={[-5, 0, -10]} />
        <Repeat count={count()}>
          {(i) => <Mesh geometry={box()} rotation={boxes()[i]} />}
        </Repeat>
      </Scene>
</SceneContext>
```
